docs/13-walkthroughs/2-deploying-doom.md

```diff
@@ -1,41 +1,147 @@
+```sh
+doug in ~ 🦄 cd src/github.com/defenseunicorns/zarf/examples/dos-games
+doug in ~/src/github.com/defenseunicorns/zarf/examples/dos-games on main 🦄
 ```
 
-### Selecting the Games Package
+2. Use the `zarf package deploy` command to deploy the zarf games package.
+```sh
+doug in ~/src/github.com/defenseunicorns/zarf/examples/dos-games on main 🦄 zarf package deploy zarf-package-dos-games-arm64.tar.zst
+
+Saving log file to
+/var/folders/bk/rz1xx2sd5zn134c0_j1s2n5r0000gp/T/zarf-2023-03-20-13-11-30-1741110763.log
+  ✔  Loading Zarf Package zarf-package-dos-games-arm64.tar.zst
 
-Since we did not provide the path to the package as an argument to the `zarf package deploy` command, Zarf will prompt you asking for you to choose which package you want to deploy. There is a useful tab-suggestions feature that makes selecting between different packages in your directories easier.
+kind: ZarfPackageConfig
+metadata:
+  name: dos-games
+  description: Simple example to load classic DOS games into K8s in the airgap
+  architecture: arm64
+build:
+  terminal: DougMBP
+  user: doug
+  architecture: arm64
+  timestamp: Mon, 20 Mar 2023 13:11:05 -0400
+  version: v0.24.3
+  migrations:
+  - scripts-to-actions
+components:
+- name: baseline
+  required: true
+  manifests:
+  - name: multi-games
+    namespace: zarf
+    files:
+    - manifests/deployment.yaml
+    - manifests/service.yaml
+  images:
+  - defenseunicorns/zarf-game:multi-tile-dark
+This package has 1 artifacts with software bill-of-materials (SBOM) included. You can view them now
+in the zarf-sbom folder in this directory or to go directly to one, open this in your browser:
+/Users/jason/src/github.com/defenseunicorns/zarf/examples/dos-games/zarf-sbom/sbom-viewer-defenseunicorns_zarf-game_multi-tile-dark.html
+
+* This directory will be removed after package deployment.
+
+? Deploy this Zarf package? (y/N)
+```
+
+3. If you do not provide the path to the package as an argument to the `zarf package deploy` command, Zarf will prompt you asking for you to choose which package you want to deploy. You can use the `tab` key, to be prompted for avaiable packages in the current working directory.
 
 ![Package Deploy Selection Tab](../.images/walkthroughs/package_deploy_tab.png)
 By hitting 'tab', you can use the arrow keys to select which package you want to deploy. Since we are deploying the games package in this walkthrough, we will select that package and hit 'enter'.
 
 ![Package Deploy Tab Selection](../.images/walkthroughs/package_deploy_tab_selection.png)
-As we have seen a few times now, we are going to be prompted with a confirmation dialog asking us to confirm that we want to deploy this package onto our cluster.v
+As we have seen a few times now, we are going to be prompted with a confirmation dialog asking us to confirm that we want to deploy this package onto our cluster.
+
+```sh
+doug in ~/src/github.com/defenseunicorns/zarf/examples/dos-games on main 🦄 zarf package deploy
```

Comment:
While visually interesting (especially for video) I find the full path distracting in a walkthrough (though I am very used to the `$` and `#` convention in wiki docs).  Something like this may be easier to read:

```suggestion
$ zarf package deploy
```